pareto/tests/test_strategic_model.py

```diff
@@ -402,6 +403,42 @@ def test_basic_reduced_build_capex_distance_based_capacity_calculated(
     assert isinstance(m.PipelineExpansionCapEx, pyo.Constraint)
 
 
```

Comment:
The existing tests  take more than an hour to complete. To test the different pipeline config options, I suggest we also use the scaled model

Reply:
@NienkeWagenaar are you referring to the tests `test_basic_build_capex_distance_based_capacity_input` `test_basic_build_capex_distance_based_capacity_calculated`, etc?. If that's the case, I'm afraid the scaling won't help because those tests only check if the components have been built. Scaling helps only if the model is solved, for example in `test_run_reduced_strategic_model`